README.md

```diff
@@ -29,17 +29,27 @@ An early prototype of HoneyBadger (v1) can be seen in the presentation "[Hide an
 4. Initialize the database. The provided username and password will become the administrator account.
 
     ```
-    $ python
+    $ python3
```

Comment:
As long as you specify Python 3 as a requirement, this shouldn't be necessary. It's an environmental configuration that prevents the standard `python` command from working properly. Updating the notification was fine, but let's leave these alone.

Reply:
Ok. I can go back and fix that quick. I'll have more small additions on my PR for the powershell utility fix.

Reply:
I've changed these back to `python`.